src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml

```diff
@@ -203,11 +203,11 @@
     </div>
 </div>
 
-<table class="table table-hover">
+<table class="table table-hover table-striped">
```

Comment:
I have already deployed similar change in APIView production now. Please let me know if that is sufficient or if you would like me to revert it,

Reply:
It's probably better to use the striping from bootstrap rather than rolling something custom